Gemfile

```diff
@@ -1,12 +1,12 @@
 source 'https://rubygems.org'
 
-gem 'rails', '~> 3.2.20'
 gem 'passenger'
+gem 'rails', '~> 3.2.20'
 
 gem 'rails-i18n', '~> 3.0.0'
 # run with `bundle install --without production` or `bundle install --without mysql` to exclude this
 group :mysql, :production do
-  gem 'mysql2', '~> 0.3.20'
+  # gem 'mysql2', '~> 0.3.20'
```

Comment:
Hi, was there a reason to disable this? Perhaps you didn't mean to check this one in?